Scripts/assignment/assignment_period.py

```diff
@@ -476,7 +476,10 @@ def _extract_timecost_from_gcost(self, ass_class):
         gcost = self._get_matrix("gen_cost", ass_class)
         cost = self._get_matrix("cost", ass_class)
         dist = self._get_matrix("dist", ass_class)
-        time = gcost - vot_inv*(cost + self.dist_unit_cost*dist)
+        time = gcost - vot_inv*self.dist_unit_cost*dist
```

Comment:
In truck assignment, `vot_inv*self.dist_unit_cost` is not used as-is due to historical reasons; in Helmet 2 it used to be hard-coded as 0.2 min/km and no one has thought of changing it since, so it is still hard-coded here: 

https://github.com/HSLdevcom/helmet-model-system/blob/4939e0505b8bde26f53116c89c31ff747cf74c45/Scripts/assignment/datatypes/car_specification.py#L40-L42

What we also *should* do is to define an own `dist_unit_cost` for trucks and trailer trucks and replace 0.2 above by `vot_inv*dist_unit_cost`. This distance unit cost should presumably be a bit higher than for private cars. This solution will change route choices somewhat.

A quick fix to get `time` matrix calculation right for trucks and trailer trucks would be to set `dist_unit_cost` for trucks and trailer trucks so that `vot_inv*self.dist_unit_cost` results in 0.2.

Reply:
Thank you for the details! Now I think I got it!